de/model.jl

```diff
@@ -12,6 +12,15 @@ function get_data(path_RNAseq)
     return Matrix(exp)
 end
 
+" Initialize the parameters based on the data. "
+function initialize_params(exp)
+    epss = exp[:, 84] ./ 100
```

Comment:
Should this be divided by 10 and not 100?

Reply:
I have run the fitting process with the updated version of model.jl. And it seems epss should be in the range of [0.0, 10.0]

Reply:
But I thought we were setting `epss` such that the initial amounts equal that of the control condition? The initial amounts are `epss / alpha`, and `alpha` is set to `0.1`. Therefore, rearranging `epss = x_0 * alpha` or `epss = x_0 * 10`.

Reply:
okay, I see